src/main/kotlin/no/skatteetaten/aurora/boober/utils/RouteUtils.kt

```diff
@@ -0,0 +1,24 @@
+package no.skatteetaten.aurora.boober.utils
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.module.kotlin.convertValue
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.fabric8.openshift.api.model.Route
+
+fun routeFromJson(jsonNode: JsonNode): Route = jacksonObjectMapper().convertValue(jsonNode)
```

Comment:
Kunne denne vært en extension function to JsonNode?

Reply:
Vi har mange andre som er i det samme mønsteret så derfor har jeg laget den slik men vi kan jo teste så ser vi hvordan det blir

Reply:
Hva med denne?
`inline fun <reified T : Any> JsonNode.read(): T = jacksonObjectMapper().convertValue(this)`

Reply:
jeg syns det siste alternativet ditt er fint. foretrekker det fremfor en funksjon med input, hva tenker du?